client/src/components/pages/home/SocialCard.js

```diff
@@ -0,0 +1,36 @@
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { Typography } from "antd";
+import Text from "antd/lib/typography/Text";
+import React, { useState } from "react";
+
+import "./SocialCard.css";
+
+/**
+ * Displays a card that links to a social media page.
+ */
+export default function SocialCard({
```

Comment:
this component should have a `cursor: pointer` when hovering it

it also might look nice to have a slight styling change when hovered, e.g.
```
.SocialCard-card:hover {
  opacity: 0.85;
  transition: 0.25s;
}
```